ApiBundle/Controller/ContentController.php

```diff
@@ -101,10 +101,32 @@ public function showOrCreateAction(Request $request, $contentId)
     public function listAction(Request $request)
     {
         $contentType = $request->get('content_type');
+        $columns = $request->get('columns');
```

Comment:
Yes but this method take orther parameter. for instance in the contentController the method for pagination take a parameter contentType.